checkov/common/models/consts.py

```diff
@@ -1,2 +1,3 @@
 SUPPORTED_FILE_EXTENSIONS = [".tf", ".yml", ".yaml", ".json", ".template"]
 ANY_VALUE = "CKV_ANY"
+DOCKER_IMAGE_REGEX = r'(?:[^\s\/]+/)?([^\s:]+)(?::)?([^\s]*)'
```

Comment:
@tronxd To be simpler, can be `:?` instead of `(?::)?`

Reply:
Absolutely!